src/components/team-member-entries/TeamMemberEntries.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useContext, useState } from "react";
+
+import { Modal } from "../modal/Modal";
+import { TeamMemberEntry } from "../team-member-entry/TeamMemberEntry";
+import { TeamMemberForm } from "../team-member-form";
+import * as Types from "../../types/types";
+import { TeamContext } from "../context/TeamContextProvider";
+
+import * as Styled from "./TeamMemberEntries.styled";
+
+export const TeamMemberEntries = ({ isModalActive, handleModal }: Types.AtBuildTeamProps) => {
+  const { teamMembers, setTeamMembers } = useContext(TeamContext);
+  const [selectedClient, setSelectedClient] = useState("");
+
+  const filterClients = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setSelectedClient(event?.target.value);
+  };
+
+  const onlyClients = teamMembers.map((entry) => entry.client);
```

Comment:
For now this solution is more than adequate ✨ , teamMembers wil exist because you get it directly from useContext. But if you are working in a different project outside of the Next.js framework without something like getStaticProps(), for example a react-native project you would put this logic inside a useEffect hook that is subscribed to the teamMembers variable.